src/app/collection/pages/collection-ncyte/header-ncyte/header-ncyte.component.html

```diff
@@ -0,0 +1,21 @@
+<header class="header">
+    <div class="ncyte-title">
+        <img class = "logo" src="../../../../assets/images/collections/ncyte.png" alt="Ncyte Center">
+         <h2 class = "logo-header">The National Cybersecurity Training
+            & Education (NCyTE) Center Curricular Resources
+        </h2>
+    </div>
+    <div class="section-browse">
+        <section class = "section-accent">
+            <i class="far fa-user-graduate icon"></i>
+            <p>
+                Developing strong cybersecurity programs and faculty
+                is key to developing a stronger cybersecurity workforce
+            </p>
+                <a href = "https://clark.center/browse?text=ncyte&currPage=1&limit=10&collection=ncyte" class="button ncyte">Browse Curriculum
+                    <i class="fas fa-arrow-right"></i>
```

Comment:
Shouldn't you use a button tag with the function click?

Reply:
I don't know what the visual is for the link clicking. But I would still consider wrapping the link with a button tag that way the whole container has the clicking functionality rather than just the words of the link.